src/presentational/button-style.ts

```diff
@@ -0,0 +1,72 @@
+import { LitElement, html, css } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import '@internetarchive/icon-donate/icon-donate.js';
+
+@customElement('iaux-button-style')
+export class MonthlyGivingCircle extends LitElement {
```

Comment:
Seems something is wrong here. your class name and custom-element not matching. Also do we really need to defined sub-component with `iaux-` prefix?

Also can we use our ia-button.ts style file from collection-browser? pretty updated that file.

Reply:
i don't need collection browser here so i will not import. i think we want to go with new button component, i am making here.